openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -13,12 +13,70 @@
 from django.dispatch import Signal
 from django.utils.http import cookie_date
 
+from edx_rest_framework_extensions.auth.jwt import cookies as jwt_cookies
 from openedx.core.djangoapps.user_api.accounts.utils import retrieve_last_sitewide_block_completed
 from student.models import CourseEnrollment
 
+from .waffle import JWT_COOKIES_FLAG
```

Comment:
Where does this import come from?

Reply:
Do you mean I should expand it and not use the explicit-dot short-hand notation?  If so, yes, I updated it.